packages/xo-server-cloud/src/index.js

```diff
@@ -106,8 +108,8 @@ class XoServerCloud {
 
   // ----------------------------------------------------------------
 
-  async _getNamespaceCatalog(namespace) {
-    const namespaceCatalog = (await this._getCatalog())[namespace]
+  async _getNamespaceCatalog(namespace, filters) {
+    const namespaceCatalog = (await this._getCatalog(filters))[namespace]
```

Comment:
`this._getCatalog` param is incorrect.

Reply:
I'm passing it here -> https://github.com/vatesfr/xen-orchestra/pull/4442/files/d7bcdfac1962869540834cb7b0e98b4e4cbe114c#diff-36046affbcce191ff1a6526d676a582fR131 I don't think it's incorrect !

Reply:
I believe it is, what's the signature of `this._getCatalog`?

Reply:
`async _getCatalog({ filters } = {}) {`

Reply:
My bad, it's correct, but it's very misleading :wink: